### ISSUE TITLE
Prerequisites for example publishing to PowerShell Gallery

### ISSUE THREAD

**Issue by johlju** [PowerShell/DscResource.Tests#252]
The following commands need to be run prior to using any PowerShellGet cmdlets. This is so that the dependency works correctly. We should be able to add these to the function `Invoke-AppveyorInstallTask`, which is run in the AppVeyor install phase, and as the first thing that is run in that function. 

```
Get-PackageProvider -Name NuGet -ForceBootstrap
Install-Module -Name PowerShellGet -Force -Repository PSGallery
```

**Comment by johlju** [comment]
Prior to this change the version of PowerShellGet was 1.0.0.1. After this change the version installed is currently 1.6.6. 

```
ModuleType Version    Name
---------- -------    ----
Script     1.6.6      PowerShellGet
Script     1.0.0.1    PowerShellGet
```

**Comment by johlju** [comment]
This was merged in PR #253.
